Riot/Modules/Analytics/PostHogAnalyticsClient.swift

```diff
@@ -62,4 +63,8 @@ class PostHogAnalyticsClient: AnalyticsClientProtocol {
         postHog?.screen(event.screenName.rawValue, properties: event.properties)
     }
     
+    func updateUserProperties(_ userProperties: AnalyticsEvent.UserProperties) {
+        // As user properties overwrite old ones via $set, compactMap the dictionary to avoid resetting any missing properties
+        postHog?.capture("$identify", properties: ["$set": userProperties.properties.compactMapValues { $0 }])
```

Comment:
Where can I find out more about what this is doing? (e.g. where is `$identity` defined)

Reply:
Cool and I assume the `$set` is also some PostHog specific syntax

Reply:
Correct. `$set` is [officially documented](https://posthog.com/docs/integrate/client/ios#setting-user-properties-via-an-event) and so seems reasonable to leave implemented like this.